src/services/observation.service.ts

```diff
@@ -426,3 +426,21 @@ export const axGetUserLifeList = async (userId, type, params) => {
     return { success: false, data: [] };
   }
 };
+
+export const axGetObsListData = async (
```

Comment:
pls remove the entire code under axGetObsListdata, as it is a duplicate of "axGetListData"